3684/1/info_taker.rb

```diff
@@ -0,0 +1,47 @@
+require 'open-uri'
+require 'nokogiri'
+
+# rubocop:disable Security/Open
+class InfoTaker
+  attr_reader :adress
+
+  def initialize(adress)
+    @adress = adress
+    @document = Nokogiri::HTML(open(adress.to_s))
```

Comment:
you do `Nokogiri::HTML(open())` at least twice
should be moved to separate method